### DISCUSSION TITLE
Improve "Definition Preview Hover" rendering and layout (similiar to JetBrains IDEs)

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/pylance-release#2939] [Enhancement]
I'm almost done with migration from PyCharm to VSCode, but a few things are holding me back! One of them being the layout and the rendering for "Definition Preview Hover" or the equivalent for "Quick Documentation Popup"  on JetBrains' IDEs.

Rather than trying to explain it in words, I rather attach a screenshot from both VSCode and PyCharm displaying the documentation for `sort_values()` method:

- ### PyCharm
![Screenshot from 2022-06-15 01-49-43](https://user-images.githubusercontent.com/18480836/173692750-ffa5a23a-f9f7-453a-80c5-2d547ccc20d0.png)

- ### VSCode
![Screenshot from 2022-06-15 01-50-53](https://user-images.githubusercontent.com/18480836/173692685-afa511bc-4928-4479-beb4-3e0d2a4ea1a2.png)

As you can see in the images above, PyCharm's definition doc preview is displaying very nicely, properly formatted and easy to read while VSCode started with all of the overload examples of the method and even when you scroll down past the overloads you see this hard to read and poorly formatted text:
![Screenshot from 2022-06-15 02-04-47](https://user-images.githubusercontent.com/18480836/173693606-eba46dc3-f390-4a70-bc3e-38ba3ded2ac9.png)

I'm not sure if this is the right repository for this issue, if it's not let me know the right one.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
